### PR TITLE
added description for DRI1 and RI1

### PR DESCRIPTION
**Pull request by frankschae**
I also started with the recommended method part.. Is there a way to preview the final look of the website?

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
PreviewPR will make one.

File: docs/src/solvers/sde_solve.md [hunk 1]
```diff
@@ -103,6 +108,8 @@ Orders are given in terms of strong order.
   with an error estimator based on Lamba due to Rackauckas. Strong order 0.5 in
   the Stratonovich sense. Can handle all forms of noise, including non-diagonal,
   scalar, and colored noise.†
+- `SimplifiedEM` - A simplified Euler-Maruyama method with weak order 1.0 and fixed step
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md:111]: move this to the weak order methods

File: docs/src/solvers/sde_solve.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ In addition, the option `symplectic=true` will turns these methods into an
 implicit Midpoint extension which is symplectic in distribution but has an
 accuracy tradeoff.
 
+If only an estimation for the expectation value of the solution is required, i.e.,
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md:39]:
```suggestion
If only an estimation for the expectated value of the solution is required, i.e.,
```

File: docs/src/solvers/sde_solve.md [hunk 3]
```diff
@@ -36,6 +36,11 @@ In addition, the option `symplectic=true` will turns these methods into an
...
+if one is only interested in an accurate draw from the distribution induced by
+a given SDE, the use of high weak order solvers is recommended.
+Specifically, `DRI1` is preferred for a high number of Wiener processes.
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md:42]: you'll want to note that none are currently adaptive.

File: docs/src/solvers/sde_solve.md [hunk 4]
```diff
@@ -236,6 +243,13 @@ The following methods require analytic derivatives of the diffusion term.
   The default settings for the drift implicitness is `theta=0.5` and
   the diffusion implicitness is `eta=0.5`.  
 
+#### High Weak Order Methods
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md:246]: note that these are not adaptive

File: docs/src/solvers/sde_solve.md [hunk 5]
```diff
@@ -245,6 +244,10 @@ The following methods require analytic derivatives of the diffusion term.
 
 #### High Weak Order Methods
 
+None of the following methods is adaptive.
```
Comment (ChrisRackauckas) [inline docs/src/solvers/sde_solve.md:247]:
```suggestion
Note that none of the following methods are adaptive.
```

Comment (frankschae) [reply docs/src/solvers/sde_solve.md]: learned something about English here ... :P

**Pull request merged**